test/stack/20-hdfs.yaml

```
@@ -0,0 +1,67 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: simple-zk
+spec:
+  image:
+    productVersion: 3.8.3
+  servers:
+    roleGroups:
+      default:
+        replicas: 1
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperZnode
+metadata:
+  name: simple-hdfs-znode
+spec:
+  clusterRef:
+    name: simple-zk
+---
+apiVersion: hdfs.stackable.tech/v1alpha1
+kind: HdfsCluster
+metadata:
+  name: simple-hdfs
+spec:
+  image:
+    productVersion: 3.3.6
+    custom: hdfs # Will be overwritten by Tilt
+    pullPolicy: IfNotPresent
+  clusterConfig:
+    dfsReplication: 1
+    zookeeperConfigMapName: simple-hdfs-znode
+    authentication:
+      tlsSecretClass: tls
```

optional, but as discussed we can do this to remove the false-positives:

```suggestion
      tlsSecretClass: tls # pragma: allowlist secret
```